vignettes/EpiNow2.Rmd

```diff
@@ -293,3 +296,137 @@ estimates$summary$summary_plot
 ```
 
 ![plot of chunk plot_regional_epinow_summary](EpiNow2-plot_regional_epinow_summary-1.png)
+
+#### When regions have different settings
```

Comment:
I'm not convinced we want to add a whole new section? Could we just demonstrate the modification workflow in the original? I think it might be better to give them different initial priors or ascertainment or something as these are good targets for domain knowledge vs changing the latent process?

Reply:
This has been reverted in f11f846e3808726bb9b92e97a78449f85f7294af.